<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowKflowSettings</class>
 <widget class="QDialog" name="windowKflowSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>kFlow Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane { 
    border: 1px solid #0b477b;
}

QTabBar::tab {
 subcontrol-origin: margin;	
 border-top-left-radius: 3px;
 border-top-right-radius: 3px;
border: 1px solid #0b477b;
border-bottom: none;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #001520, stop: 1 #000809);
padding: 4 4px;
color: white;
}
QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #FFFFFF, stop: 1 #FAFAFA);
color:black;
}

QTabBar::tab:!selected {
    margin-top: 4px; /* make non-selected tabs look smaller */
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>Simulation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="binaryPathLayout">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Binary Path:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="path">
           <property name="toolTip">
            <string>Comsumed power</string>
           </property>
           <property name="toolTipDuration">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPath">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="algirhtmLayout">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Algorithm:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Algorithm">
           <item>
            <property name="text">
             <string>Gauss-Seidel</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Newton-Rhapson</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Shirmuhamad</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="maxIterationsLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Max. Iterations:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="maxIterations"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="minErrorLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Minimum Error:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="minError"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="saveIterations">
         <property name="text">
          <string>Save Iterations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="verbose">
         <property name="text">
          <string>Verbose</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Colors</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
